com.unity.multiplayer.mlapi/Runtime/Serialization/Pooled/NetworkReaderPool.cs

```diff
@@ -9,8 +9,8 @@ namespace MLAPI.Serialization.Pooled
     /// </summary>
     public static class NetworkReaderPool
     {
-        private static byte createdReaders = 0;
-        private static readonly Queue<PooledNetworkReader> readers = new Queue<PooledNetworkReader>();
+        private static byte s_CreatedReaders = 0;
+        private static readonly Queue<PooledNetworkReader> k_Readers = new Queue<PooledNetworkReader>();
```

Comment:
I realize the coding standards say something that's private & read-only is a "k_", but to me the sprit of "k_" is a constant, single value (like when I look at code I expect it to be a scalar, immutable thing), so I would have chosen "s_"

Reply:
as a C++ developer in C# land, I do agree with you there but I simply did follow Unity C# standards naming and didn't pick my own preference over it.

Reply:
well, `private static readonly Queue<PooledNetworkReader> k_Readers;` fundamentally translates to "a constant pointer to a queue instance" here. instance is not constant but the pointer is, eh...

Reply:
oh actually, I might get rid of `readonly` keyword there (and in other places) to still be Unity C# compliant but avoid `k_` to prefer `s_` — what do you think?

Reply:
I wouldn't want to lose the readonly benefit for the same of optics.  I probably would opt for 's_', but it's not a showstopper to me by any means.